src/components/DataTable/DataTable.spec.js

```diff
@@ -0,0 +1,131 @@
+import { mount } from '@vue/test-utils'
+import DataTable from './DataTable.vue'
+
+const columns = [
+  {
+    label: 'Name',
+    value: 'name'
+  },
+  {
+    label: 'Member Count',
+    value: 'memberCount'
+  },
+  {
+    label: 'Roles',
+    value: 'roles'
+  }
+]
+
+const rows = [
+  { name: 'Staging Team', memberCount: 57, roles: 'Admin' },
+  { name: 'Data Science', memberCount: 405, roles: 'Admin' },
+  { name: 'Dev Ops', memberCount: 22, roles: 'Admin' },
+  { name: 'Production Team', memberCount: 35, roles: 'Admin' },
+  { name: 'Winter Interns', memberCount: 90, roles: 'Admin' }
+]
+
+const ascRows = [
+  { name: 'Data Science', memberCount: 405, roles: 'Admin' },
+  { name: 'Dev Ops', memberCount: 22, roles: 'Admin' },
+  { name: 'Production Team', memberCount: 35, roles: 'Admin' },
+  { name: 'Staging Team', memberCount: 57, roles: 'Admin' },
+  { name: 'Winter Interns', memberCount: 90, roles: 'Admin' }
+]
+
+describe('props', () => {
+  test('displays columns', () => {
+    const wrapper = mount(DataTable, {
+      props: {
+        rows: rows,
```

Comment:
Doesn't look like this test uses/needs rows so I wouldn't pass any in. Could use just `[]` instead. Keeps the test focused on one thing

```suggestion
        rows:[],
```